__tests__/app.test.js

```diff
@@ -118,6 +118,7 @@ describe("get /api/articles/:article_id", () => {
         expect(article.article_img_url).toBe(
           "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
         );
+        expect(article.comment_count).toBe("11");
```

Comment:
Ideally we should be asserting against this new property in a new test, we never want to be going back and editing old tests as good practice 👍